pages/[id]/index.jsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from "next/router";
+import BackButton from "@/components/BackButton";
+import styled from "styled-components";
+
+export default function GameDetails({ initialGames }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const selectedGame = initialGames.find((game) => game.id === id);
+
+  if (!selectedGame) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <>
+      <BackButton href={"/"}></BackButton>
```

Comment:
This can be self-closing and you don't need the curly brackets, since the path is a normal string:

```suggestion
      <BackButton href="/" />
```